apps/licences/src/pages/licences/[licenseNumber].tsx

```diff
@@ -61,44 +61,52 @@ export default function LicenceDetails() {
     () => [
       {
         key: "dataAttribuzionePunteggio",
-        render: (_, item) => (
-          <>
-            <Typography
-              color="text.secondary"
-              fontWeight="regular"
-              textAlign="initial"
-              variant="body1"
-            >
-              {new Date(item.dataAttribuzionePunteggio).toLocaleDateString()}
-            </Typography>
-            {item.codiceVerbale ? (
-              <Link
-                color="text.primary"
-                component="button"
-                onClick={() => onDetailClicked?.(item)}
-                sx={{
-                  textAlign: "left",
-                  textDecoration: "underline",
-                }}
-              >
-                {item.descrizioneEventoPunteggio}
-              </Link>
-            ) : (
+        render: (_, item) => {
+          const description = item.descrizioneEventoPunteggio?.toLowerCase();
+          return (
+            <>
               <Typography
                 color="text.secondary"
-                fontWeight="medium"
+                fontWeight="regular"
                 textAlign="initial"
                 variant="body1"
               >
-                {item.descrizioneEventoPunteggio}
+                {new Date(item.dataAttribuzionePunteggio).toLocaleDateString()}
               </Typography>
-            )}
-          </>
-        ),
+              {description === "verbale" ? (
```

Comment:
I think it's too weak as a control condition.
I think it's better to rely on the presence of additional _(optional)_ information that will be shown in the modal

Reply:
What if we control the event type code as condition? I think it could be good enough and more clear to read since all eventTypes are mapped as Enum.